scripts/infinity_grid.py

```diff
@@ -100,8 +103,10 @@ def apply_prompt_replace(p, v):
     replace = val[1].strip()
     if Script.VALIDATE_REPLACE and match not in p.prompt and match not in p.negative_prompt:
         raise RuntimeError(f"Invalid prompt replace, '{match}' is not in prompt '{p.prompt}' nor negative prompt '{p.negative_prompt}'")
-    p.prompt = p.prompt.replace(match, replace)
-    p.negative_prompt = p.negative_prompt.replace(match, replace)
+    if p.contains("negative"):
```

Comment:
This seems to break the existing Prompt Replace behavior of replacing in either prompt

Reply:
`p.contains` doesn't even make sense for that matter.